packages/react-components/src/components/Select/Select.css

```diff
@@ -9,6 +9,8 @@
 
 /* Label above select input */
 .bcds-react-aria-Select--Label {
+  display: flex;
```

Comment:
Let's revert the changes inside of `.bcds-react-aria-Select--Label` and its children. We want this label to keep its default `display: inline` behaviour. If we were to set this to `display: flex`, things break when text wraps, and the "(optional)" chunk starts floating next to the label.

Current `main`:
<img width="1840" alt="Existing Select with very long label" src="https://github.com/user-attachments/assets/6f7d572b-8001-40b4-907d-3afe93577910">

Proposed in this branch:
<img width="1840" alt="Screenshot 2024-07-17 at 1 16 54 PM" src="https://github.com/user-attachments/assets/4fd384c2-1b98-43c3-9baa-af88045faa12">

We originally had a `<span>` around the "(optional)" chunk because it was used to color the chunk a different shade than the main label. If we are no longer doing that, we can simplify the markup in the component code and remove the extra span, and just build a single string inside of `<Label>`. However, we don't have a current copy of the design for this component that uses the "(optional)" chunk to confirm that this is the requirement.

Reply:
fixed in [f415819](https://github.com/bcgov/design-system/pull/390/commits/f415819499ddec73f9dafffa279e984238970856)